setup.py

```diff
@@ -0,0 +1,42 @@
+import setuptools
+
+with open('README.md', 'r') as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name='robosat',
+    version='1.2.0',
+    author='Mapbox',
+    description='Semantic segmentation on aerial and satellite imagery.',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    url='https://github.com/mapbox/robosat',
+    packages=setuptools.find_packages(),
+    classifiers=(
+        'Programming Language :: Python :: 3',
+        "License :: OSI Approved :: MIT License",
+        'Operating System :: OS Independent',
+    ),
+    install_requires=[
+        'flask>=1.0',
+        'geojson>=2.4',
+        'matplotlib>=3.1',
+        'mercantile>=1.0',
+        'numpy>=1.16',
+        'opencv-python>=4.0',
+        'opencv-contrib-python>=4.0',
+        'osmium>=2.15',
+        'pillow>=6.0',
+        'pyproj>=2.1',
+        'rasterio>=1.0',
+        'requests>=2.22',
+        'rtree>=0.8',
+        'scipy>=1.3',
+        'shapely>=1.6',
+        'supermercado>=0.0.5',
+        'toml>=0.10',
+        'torch>=1.1',
+        'torchvision>=0.3',
+        'tqdm>=4.32',
```

Comment:
How did you come up with this list and the versions? List from `requirements.in` and then manually getting the versions from the `requirements.txt` lockfile? Then why is it `>=` above? Can we somehow automate this by parsing the requirements files? Otherwise updating dependencies gets even more complicated than it is already right now.

Reply:
Yep exactly. And I did not want to specify exact versions as the solver will likely need to some flexibility for installation into a larger project. As an alternative to `>=`, we could specify `~=` to enforce a compatible version

Reply:
I updated the requirements to enforce compatible versions with `~=`, except for `osmium==2.15.2`, which requires an exact version due a known issue with their latest release
https://github.com/osmcode/pyosmium/issues/123

Automatically parsing the `requirements.txt` is possible, but would need to update for less strict versioning, and is generally not advised